src/main/java/sixthtask/ThreeArrays.java

```diff
@@ -0,0 +1,36 @@
+package sixthtask;
+
+import utils.*;
+import java.io.*;
+import java.util.*;
+
+public class ThreeArrays {
+    private static final String INPUT_FILE = "src/main/resources/input_files/input_sixthtask.txt";
+    private static final String OUTPUT_FILE = "target/logs/output_sixthtask.txt";
+
+    public static void main(String[] args) throws IOException {
+        Scanner scanner = new Scanner(new File(INPUT_FILE));
+        String line;
+        List<String> stringList = new ArrayList<>();
+        while (scanner.hasNextLine()) {
+            line = scanner.nextLine();
+            stringList.add(line);
+        }
+        scanner.close();
+        List<int[]> intList = new ArrayList<>();
+        for (String s : stringList) {
+            String[] strings = s.trim().split("\\s+");
+            int[] nums = new int[strings.length];
+            for (int j = 0; j < strings.length; j++) {
+                nums[j] = Integer.parseInt(strings[j]);
+            }
+            intList.add(nums);
+        }
+        int[] firstArr = intList.get(0);
+        int[] secondArr = intList.get(1);
+        int[] thirdArr = intList.get(2);
+        int[] initialResult = ArrayUtils.mergeTwoSortedArrays(firstArr, secondArr);
```

Comment:
there is no need in the declared above arrays
you can call them when you call the merge method
like: ArrayUtils.mergeTwoSortedArrays(list.get(X), list.get(Y);

Reply:
Right.
I've made the changes.